frontend/src/pages/CoursesList/CoursesContainer.tsx

```diff
@@ -3,49 +3,56 @@ import { useState } from 'react';
 import { ICourse } from 'types/course';
 import { useApplyCourse, useGetPaginatedCourses } from 'api/courses';
 import { useGetProfile } from 'api/profile';
-import { useFetchNextPage } from 'hooks';
+import { useFetchNextPage, useGetCoursesFilters } from 'hooks';
 import { Role } from 'constants/menuRoles';
 import { getWindowLabelByWidth } from 'utils/helpers/getWindowLabelByWidth';
 
 import CoursesList from './CoursesList';
 
 const CoursesContainer: React.FC = () => {
-  const { mutate, isLoading } = useApplyCourse();
+  const [targetId, setTargetId] = useState<string | undefined>();
+
+  const { mutate, isLoading: isApplyCourseLoading } = useApplyCourse();
```

Comment:
should we add mutate name here?

Reply:
yep, its true, it was before my changes) I fixed.